project-base/app/.env.test

```diff
@@ -5,10 +5,15 @@ DATABASE_NAME=shopsys-test
 MAILER_DSN=null://null
 MESSENGER_TRANSPORT_DSN=in-memory://
 
-GOPAY_IS_PRODUCTION_MODE=false
-GOPAY_EN_GOID=test-data
-GOPAY_EN_CLIENTID=test-data
-GOPAY_EN_CLIENTSECRET=test-data
-GOPAY_CS_GOID=test-data
-GOPAY_CS_CLIENTID=test-data
-GOPAY_CS_CLIENTSECRET=test-data
+GOPAY_CONFIG='[
+    {
+        "goid": "test-data",
+        "clientId": "test-data",
+        "clientSecret": "test-data",
+        "isProductionMode": false,
```

Comment:
:question: 

What about setting one domain to production and second domain to testing mode ? Will be there such a scenario ?

Reply:
I believe so... for example when the first domain is running and the second will is still in development :) 

when the GoPay configuration data should be the same for the first and second domain it's a little cumbersome, but still doable

```
GOPAY_CONFIG='[
    {
        "goid": "<goid1>",
        "clientId": "<clientId1>",
        "clientSecret": "<clientSecret1>",
        "isProductionMode": true,
        "domains": [
            1
        ]
    },
    {
        "goid": "<goid1>",
        "clientId": "<clientId1>",
        "clientSecret": "<clientSecret1>",
        "isProductionMode": false,
        "domains": [
            2,3
        ]
    }
]'
```